src/pages/Explore/Explore.jsx

```diff
@@ -0,0 +1,16 @@
+import { useSelector } from "react-redux";
+import { Post } from "../../components";
+export const Explore = () => {
+  const { posts } = useSelector((store) => store.posts);
+  return (
+    <>
+      <div className="section">
+        <div>
+          {posts.map((post) => {
+            return <Post post={post} key={post._id} />;
+          })}
+        </div>
+      </div>
+    </>
```

Comment:
The empty fragment is not needed here. Also check if you really need two `div` tags or not